README.md

```diff
@@ -80,4 +80,11 @@ or
 $ poetry run pytest tests/test_file_name 
 
 
+## ANSIBLE
 
+Log to your Control Node and use this command to provision host VM:
+
+
+$ ansible-playbook Playbook -i Inventory --ask-vault-pass
```

Comment:
If you have a vault, you could commit it to the repository, as the values are encrypted in the vault file. You just need to keep your vault password secret.

But not a big deal.